mdms/mdms-tools/pom.xml

```diff
@@ -29,6 +48,11 @@
             <groupId>de.metanome</groupId>
             <artifactId>backend</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.antlr</groupId>
+            <artifactId>antlr4-runtime</artifactId>
+            <version>4.3</version>
```

Comment:
Versions of dependencies should be managed in the parent `pom.xml` to avoid disagreeing dependency specifications. For that matter, ANTLR might already be in use in the `mdms-client` module.
You can add the dependency in the parent `pom.xml` within the `<dependencyManagement>` block and then remove the `<version>` tag right here.